contracts/deploy-tezos-swap.js

```diff
@@ -0,0 +1,133 @@
+const conseiljs = require("conseiljs");
+const config = require("./config.json");
+const tezosConfig = require("./tezos/build/TezosSwap/step_000_cont_1_contract.json");
+const log = require("loglevel");
+const conSign = require("conseiljs-softsigner");
+const fetch = require("node-fetch");
+const Web3 = require("web3");
+
+const init = async () => {
+  const logger = log.getLogger("conseiljs");
+  logger.setLevel("error", false);
+  const store = {};
+  store.keyStore = await conSign.KeyStoreUtils.restoreIdentityFromSecretKey(
+    config.tezos.privateKey
+  );
+  store.signer = await conSign.SoftSigner.createSigner(
+    conseiljs.TezosMessageUtils.writeKeyWithHint(
+      store.keyStore.secretKey,
+      "edsk"
+    )
+  );
+  //   console.log(store.keyStore);
+  conseiljs.registerFetch(fetch);
+  conseiljs.registerLogger(logger);
+  const web3 = new Web3(new Web3.providers.HttpProvider(config.ethereum.RPC));
+  return { store, web3 };
+};
+
+const estimateFees = async (store, web3) => {
+  try {
+    const fa12Estimate =
+      await conseiljs.TezosNodeWriter.testContractDeployOperation(
+        config.tezos.RPC,
+        config.tezos.chain_id,
+        store.keyStore,
+        0,
+        undefined,
+        100000,
+        10000,
+        20000,
+        trim(tezosConfig),
+        conseiljs.TezosLanguageUtil.translateMichelsonToMicheline(
+          `(Pair (Pair False (Pair "${store.keyStore.publicKeyHash}" 0)) (Pair (Pair {} 15) (Pair {} 0)))`
```

Comment:
better to use named constants

Reply:
Actually this is a generic script, the storage can be changed too that's why I left it as it is